java/build.gradle

```diff
@@ -12,8 +12,30 @@ repositories {
 }
 
 dependencies {
+    implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.5'
+    implementation 'commons-io:commons-io:2.6'
     implementation 'org.junit.platform:junit-platform-surefire-provider:1.0.0'
```

Comment:
This is not related to the latest changes in this commit, but shouldn't surefire be a testImplementation dependency?

Reply:
Fixed it